src/main/resources/biz_de.properties

```diff
@@ -241,6 +241,8 @@ DeleteTenantJobFactory.label = Mandant löschen
 DeleteTenantJobFactory.simulate = Löschen Simulieren
 DeleteTenantJobFactory.simulate.help = Ist 'Ja' ausgewählt, wird das Löschen nur simuliert. Dies kann Ihnen helfen, die Auswirkungen des Löschens zu verstehen, da Sie im Protokoll sehen können, welche Daten gelöscht werden würden.
 DeleteTenantJobFactory.simulateInfo = Die Löschung wird nur simuliert.
+DeleteTenantJobFactory.takeAction = Wirklich löschen?
+DeleteTenantJobFactory.takeAction.help = Ist 'Nein' ausgewählt, wird das Löschen nur simuliert.
```

Comment:
Warum kommen deutsche Keys hinzu? Das sollte doch eigentlich nicht passieren?

Reply:
Sehr gute Frage. Ich habe mich auch gewundert.

Reply:
Naja das spricht doch dafür dass sie gelöscht wurden 😅

Reply:
Macht absolut Sinn. Aber wieso added Lokalise die dann wieder?

Reply:
Wie werden denn die gelöschten Keys normalerweise aus Lokalise entfernt???

Reply:
Vielleicht mal die Settings vom GH Plugin zwischen biz und ox abgleichen. Eigentlich darf er nur die DE Datei pullen und nicht alle

Reply:
Bei biz wird auch nur die DE-Datei gepullt....

Reply:
Naja dann versteh ichs nicht wieso er die gelöschten Keys nicht mitbekommt. Oder werden die dir in Lokalise irgendwie markiert?

Reply:
Hab es mit ner Konfig-einstellung behoben und dann im Follow-up PR gelöst: https://github.com/scireum/sirius-biz/pull/1840